examples/ct/structures.go

```diff
@@ -121,6 +121,38 @@ func SignV1SCTForPrecertificate(km crypto.KeyManager, cert *x509.Certificate, t
 	return serializeAndSignSCT(km, leaf, sctInput, t)
 }
 
+// SignV1TreeHead signs a tree head for CT. The input STH should have been built from a
+// backend response and already checked for validity.
+func SignV1TreeHead(km crypto.KeyManager, sth *ct.SignedTreeHead) error {
```

Comment:
True but I'm hoping to move it beyond toy status at some point. I think the serialization code is pretty much equivalent to the client side, and does have tests.

How about I pull this out into serialize.go, unexport it for now and we can revisit later?